server/src/services/ui/clientOptions.js

```diff
@@ -1,6 +1,10 @@
 const {
   clientSideOptions,
   map: { enableMapJsFilter },
+  map: { enablePokemonPopupCoordsSelector },
+  map: { enableGymPopupCoordsSelector },
+  map: { enablePokestopPopupCoordsSelector },
+  map: { enablePortalPopupCoordsSelector },
```

Comment:
One small thing here, instead of importing them like this, please consolidate it to be like this:
```js
const {
  clientSideOptions,
  map: { enableMapJsFilter, enablePokemonPopupCoordsSelector, ...the rest },
  api: {
    pvp: { levels },
  },
} = require('../config')
```
replacing `the rest` with the other imports.